kong/clustering/services/sync/rpc.lua

```diff
@@ -274,6 +274,32 @@ local function lmdb_delete(db, t, delta, opts, is_full_sync)
 end
 
 
+local function precheck_deltas(deltas)
```

Comment:
Maybe a more specific name

```suggestion
local function adapt_default_workspace(deltas)
```

Reply:
No, it will has other work to do (in EE)

Reply:
might be `preprocess_`, we dont check something here, just pre-loading something.

Reply:
changed.